models/attachment.go

```diff
@@ -104,7 +114,7 @@ func getAttachmentByUUID(e Engine, uuid string) (*Attachment, error) {
 	attach := &Attachment{UUID: uuid}
 	has, err := e.Get(attach)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
because a fucking retarded ask me to

https://github.com/go-gitea/gitea/pull/1700#pullrequestreview-37275893